ebuild-maintenance/removal/text.xml

```diff
@@ -83,6 +83,48 @@ Date:   Tue Oct 3 21:43:03 2017 +1100
   Closes: https://bugs.gentoo.org/629144
 </pre>
 
+</body>
+</section>
+
+<section>
+<title>Removing a virtual package</title>
+<body>
+
+<p>
+Virtual packages are generally removed when they have no more than one
+provider left. The removal is preceded by updating the remaining ebuilds
+not to use the virtual. Since virtuals do not install any files, there
+is little value in proactively forcing them to be uninstalled from user
+systems and therefore an alternative removal process is recommended.
```

Comment:
I wonder if this is applicable to all removals of virtuals? I.e. should the policy say "recommended", or merely point out that the alternative procedure exists?